libs/httpproxy/pom.xml

```diff
@@ -77,13 +77,12 @@
         <dependency>
             <groupId>junit</groupId>
             <artifactId>junit</artifactId>
-            <version>4.8</version>
             <scope>test</scope>
         </dependency>
         <dependency>
             <groupId>org.mockito</groupId>
-            <artifactId>mockito-all</artifactId>
-            <version>1.9.0</version>
+            <artifactId>mockito-core</artifactId>
```

Comment:
Any reason we're adding `mockito-core` as a dependency here? I'd expect `mockito-all` to make this unnecessary.